lib/open_api_annotator/serializer_annotatable.rb

```diff
@@ -95,6 +99,13 @@ def open_api_attributes
       def open_api_resource_name
         name.remove(/Serializer\z/)
       end
+
+      private
+
+      def skip_open_api_validation?
+        return @open_api_validation_skipped unless @open_api_validation_skipped.nil?
```

Comment:
Nitpick: I think `if val` is better for readability.

```suggestion
        return @open_api_validation_skipped if @open_api_validation_skipped
```

Reply:
If `@open_api_validation_skipped` is false, `skip_open_api_validation?` is evaluated as `OpenApiAnnotator.config.default_skip_open_api_validation`.
It is not expected behavior.

Reply:
I see. you are right.